src/components/shared/Header.tsx

```diff
@@ -0,0 +1,179 @@
+import React, { useContext, useEffect, useState } from 'react'
+import { useMediaQuery } from 'react-responsive'
+import { Link, useHistory, useLocation } from 'react-router-dom'
+import MultiColorIcon from '../../assets/images/colored-icon.svg'
+import { mobileDeviceOrTabletWidth } from '../../constants'
+import { AppContext } from '../../context'
+import { LanguageType } from '../../types'
+import Translate from '../../utils/translate/translateService'
+import { withLocaleUrl } from '../../utils/utils'
+import PartnershipsConfig from "PartnershipsConfig";
+import { Dropdown, Accordion, Icon } from "semantic-ui-react";
+
+export const Header = () => {
+  const [tab, setTab] = useState("");
+  const [state, dispatch] = useContext(AppContext);
+  const [ partnershipDrownDownActive, setPartnershipDrownDownActive ] = useState(false);
+
+  const getTabClass = (tabName: string) => {
+    return tab.includes(tabName) ? 'nav__item--active col-auto h-100 flex center-item' : 'nav__item col-auto h-100 flex center-item';
+  }
+
+  const handleClick = () => setPartnershipDrownDownActive(!partnershipDrownDownActive)
+
+  const usePageViews = () => {
+    let location = useLocation();
+    useEffect(() => {
+      setTab(location.pathname)
+    }, [location]);
+  }
+
+  const history = useHistory();
+
+  const changeLanguages = () => {
```

Comment:
we should stick this in a utils function as there are now two places where this logic is invoked right